widget/ui/src/components/Header/Header.tsx

```diff
@@ -5,24 +5,46 @@ import React from 'react';
 
 import { Typography } from '../Typography';
 
-import { Container, globalHeaderStyles, Suffix } from './Header.styles';
+import { Container, globalHeaderStyles, Prefix, Suffix } from './Header.styles';
 
 export function Header({
   prefix,
   suffix,
   title,
+  disableCurves,
+  titlePosition,
+  transparent,
```

Comment:
so with `background: "trasnparent" | undefined` we will have a prop named background which can accept only a string "transparent" or undefined. Does not it seem like a boolean?

Reply:
There are two approach here, see it as an state or see it as a color value. In first approach it will be a boolean, in second approach it can be any colors, including `transparent`.

Second approach is more future since you can add more color values without needing to change your interface, like `background="red"` or `backckground="#000"`.

You don't need change code inside `Header`, you can consider transparent as boolean. but in its interface you can consider my suggestion.

Reply:
Considering that we decided to revert changes related to `Modal` component, I entirely reverted the changes related to this prop.